<?xml version="1.0" encoding="UTF-8"?>

<launch>

  <include file = "$(find pkg_techfest_imc)/launch/gazebo.launch" />

  <include file = "$(find pkg_techfest_imc)/launch/rviz.launch" />
  <node pkg="pkg_techfest_imc" name="maze_solver" type="MazeSolver.py" output="screen"> 
  
</node>
</launch>
